src/swap/handle_swap_sign_transaction.c

```diff
@@ -4,9 +4,14 @@
 #include "swap_lib_calls.h"
 #include "swap_utils.h"
 #include "sol/printer.h"
+#include "util.h"
+
+#define MAX_SWAP_TOKEN_LENGTH 15
```

Comment:
Already defined in `handle_get_printable_amount.c`

Reply:
Done